models/documentlistmodel.cpp

```diff
@@ -191,7 +192,8 @@ int DocumentListModel::mimeTypeToDocumentClass(QString mimeType) const
                || mimeType == QLatin1String("application/vnd.openxmlformats-officedocument.presentationml.slideshow")
                || mimeType == QLatin1String("application/vnd.ms-powerpoint.presentation.macroEnabled.12")
                || mimeType == QLatin1String("application/vnd.ms-powerpoint.template.macroEnabled.12")
-               || mimeType == QLatin1String("application/vnd.ms-powerpoint.slideshow.macroEnabled.12") ) {
+               || mimeType == QLatin1String("application/vnd.ms-powerpoint.slideshow.macroEnabled.12") 
+               || mimeType == QLatin1String("text/csv")) {
```

Comment:
Presentation?

Reply:
No. I added "text/csv"

Reply:
Be it however, you added csv to be handled as a presentation format.

Reply:
Ooops, I see. I'll correct

Reply:
Done